src/mongo/user.js

```diff
@@ -1,10 +1,10 @@
-const {model, Schema} = require('mongoose');
+const { model, Schema } = require('mongoose')
 
-let usersSchema = new Schema({
-	User: {type: String},
-	Tag: {type: String},
-	Daily: {type: String, default: 0},
-	DailyTime: {type: String, default: null}
+const usersSchema = new Schema({
+  User: { type: String },
+  Tag: { type: String },
+  Daily: { type: String, default: 0 },
+  DailyTime: { type: String, default: null }
 })
 
-modole.exports = model("users", usersSchema)
\ No newline at end of file
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Shibe-Studio/Shynk/pullRequest?prid=7543351)
```suggestion
modole.exports = model("users", usersSchema)
```